sandbox/src/mocks/patientServiceSearch/requests/RcMinimal.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Isn't this example off-topic for this PR? I thought it related to the roles API>